api/refresh_continuous_aggregate.md

```diff
@@ -93,6 +99,12 @@ END
 $$;
 ```
 
+Forcing the refresh of the continuous aggregate `conditions` between `2020-01-01` and
+`2020-02-01` exclusive even if it is already refreshed.
+
+```sql
+CALL refresh_continuous_aggregate('conditions', '2020-01-01', '2020-02-01', TRUE);
```

Comment:
Probably good to encourage named parameters to avoid problems in the future.
```suggestion
CALL refresh_continuous_aggregate('conditions', '2020-01-01', '2020-02-01', force => TRUE);
```